src/config/resolve-config.js

```diff
@@ -13,6 +14,13 @@ const getExplorerMemoized = mem(opts => {
     cache: opts.cache,
     transform: result => {
       if (result && result.config) {
+        if (typeof result.config === "string") {
+          const modulePath = resolve.sync(result.config, {
+            basedir: path.dirname(result.filepath)
```

Comment:
It makes sense to me to resolve modules relative to the configuration file. Let me know if you can think of cases where this won't work.